loony_dev/agents/coding.py

```
@@ -64,6 +65,234 @@ def execute(self, task: Task) -> TaskResult:
+                    logger.warning(
+                        "Issue #%d: Coderabbit review retries exhausted",
+                        task.issue.number,
+                    )
+                    task.mark_review_exhausted()
+                    break
+
+                logger.info(
+                    "Issue #%d: Coderabbit found issues (attempt %d/%d), asking Claude to fix",
+                    task.issue.number, attempt + 1, max_review,
+                )
+                fix_stdout, fix_stderr, fix_rc = self._run_claude_cli(
```

Ruff (0.15.11)

[warning] Unpacked variable `fix_stdout` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)